website/siteConfig.js

```diff
@@ -40,6 +40,8 @@ const siteConfig = {
     primaryColor: '#004d97',
     secondaryColor: '#004d97',
   },
+  /*Google analytics tracking ID */
+  gaTrackingId: "UA-35892077-12",
```

Comment:
sorry to be a style nazi but should be single quotes for consistency 😅

Reply:
nice catch. updated it!